project-addons/partner_risk_advice/res_partner.py

```diff
@@ -20,12 +20,60 @@
 ##############################################################################
 
 from openerp import fields, models, api
+from openerp.osv import osv, fields as fields2
+from openerp.tools.translate import _
+
+WARNING_MESSAGE = [
+                   ('no-message','No Message'),
+                   ('warning','Warning'),
+                   ('block','Blocking Message')
+                   ]
+
+WARNING_HELP = _('Selecting the "Warning" option will notify user with the message, Selecting "Blocking Message" will throw an exception with the message and block the flow. The Message has to be written in the next field.')
 
 
 class res_partner(models.Model):
     _inherit = "res.partner"
 
     risk_advice_ids = fields.One2many ("partner.risk.advice", "partner_id")
+    _columns = {
+        'rma_warn': fields2.selection(WARNING_MESSAGE, 'Invoice', help=WARNING_HELP, required=True),
+        'rma_warn_msg': fields2.text('Message for RMA'),
+    }
```

Comment:
lo mismo aquí, es una clase de la nueva api, podéis definir los campos con la nueva api, lo podemos dejar así no falla, pero tenéis que ir orientándoos a lo nuevo, esto en la versión 10 ya no está permitido.